data_gen.py

```diff
@@ -0,0 +1,43 @@
+
+start_date_testing = datetime.date(2006, 2, 1)
+end_date_testing = datetime.date(2018, 2, 1)
+
+start_date_validation = datetime.date(2018, 2, 1)
+end_date_validation = datetime.date(2022, 2, 1)
+
+
+
+
+"""
+The amount of trading dates may not be the same for each two week period because of weekends, holidays
+start_date: Earliest date that data will be returned on for all data sets
+end_date: Latest date that data will be returned on for all data sets
+files_to_generate: The amount of data sets to be generated
+range: The time period in days for each data set
+Note that non trading days do not display anything.
```

Comment:
by "not display anything" does that mean null on the data or the day is skipped completely?

Reply:
The day is skipped completely